orchagent/main.cpp

```diff
@@ -140,6 +145,121 @@ void init_gearbox_phys(DBConnector *applDb)
     delete tmpGearboxTable;
 }
 
+bool getSystemPortConfigList(vector<sai_system_port_config_t> &sysportcfglist)
+{
+    DBConnector cfgDb("CONFIG_DB", 0);
+    DBConnector appDb("APPL_DB", 0);
```

Comment:
looks like we are not using it. can we remove it ?

Reply:
These are used in lines 153-155 below.